### PR TITLE
Added link to docs on components isolation

### PR DESCRIPTION
**Pull request by SteveALee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # `isolate()`
 
-> A utility function to make scoped dataflow components in Cycle.js.
+> A utility function to make scoped dataflow components in Cycle.js. For example, this isolates DOM events seen by a component to those from it's own nodes, not the entire tree. 
+
+See the cycle.js [documentation on components](http://cycle.js.org/components.html#multiple-instances-of-the-same-component) for further details.
```
Comment (staltz) [inline README.md]: And here, `cycle.js` => `Cycle.js`.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # `isolate()`
 
-> A utility function to make scoped dataflow components in Cycle.js.
+> A utility function to make scoped dataflow components in Cycle.js. For example, this isolates DOM events seen by a component to those from it's own nodes, not the entire tree.
```
Comment (staltz) [inline README.md]: Thanks for the PR. :)
I'd like to keep this headline very short. Can you remove the second sentence?
Then I'd merge it.

Comment (SteveALee) [inline README.md]: Sure but I felt it need a bit of context as 'scoped dataflow components' is very abstract and I could not figure out what it did till someone pointed out the docs. And then I found the video. I came here from seeing Isolate in the examples and wanted to know what it was about.

**Comment by staltz** [comment]
Thanks!

**Pull request merged**